src/main/java/address/updater/UpdateManager.java

```diff
@@ -107,6 +109,7 @@ private void checkForUpdate() {
         if (latestVersion.get().isEarlyAccess() != MainApp.IS_EARLY_ACCESS) {
             raise(new UpdaterFinishedEvent(MSG_DIFF_CHANNEL));
             logger.debug(MSG_DIFF_CHANNEL);
+            //TODO: how could this happen?
```

Comment:
We'll, logging a debug message (which won't even be enabled) is not the way to detect we've messed up the update :-p